onnxruntime/python/tools/transformers/models/stable_diffusion/benchmark.py

```diff
@@ -315,29 +315,25 @@ def get_optimum_ort_pipeline(
                 directory,
                 provider=provider,
                 session_options=None,
-                use_io_binding=False,
```

Comment:
cc @Tianleiwu I'm not sure if it is intended to always not use iobinding, even for gpu. Please let me know in case it is intended and I can revert this change with comment.

Reply:
https://github.com/huggingface/optimum/blob/fd47a73267c3a71ea4e3c02f92260ae61c5ae372/optimum/onnxruntime/modeling_diffusion.py#L352-L355

Reply:
Found reason to be unsuported by optimum.